src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -2,18 +2,45 @@
 
 /**
  * <p>На игровом поле находится робот. Позиция робота на поле описывается двумя
- * целочисленным координатами: X и Y. 
- * Ось X смотрит слева направо, ось Y — снизу вверх.(Помните, как рисовали 
+ * целочисленным координатами: X и Y.
+ * Ось X смотрит слева направо, ось Y — снизу вверх.(Помните, как рисовали
  * графики функций в школе?)
  * В начальный момент робот находится в некоторой позиции на поле.
  * Также известно, куда робот смотрит: вверх, вниз, направо или налево.
  * Ваша задача — привести робота в заданную точку игрового поля.
  * Реализуйте метод moveRobot()</p>
  */
+
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        if (robot.getX() < toX) {
+            while (robot.getDirection() != Direction.RIGHT) {
+                robot.turnRight();
+            }
+        }
+        if (robot.getX() > toX) {
+            while (robot.getDirection() != Direction.LEFT) {
+                robot.turnLeft();
+            }
+        }
+        while (robot.getX() != toX) {
+            robot.stepForward();
+        }
+        if (robot.getY() < toY) {
+            while (robot.getDirection() != Direction.UP) {
+                robot.turnRight();
+            }
+        }
+        if (robot.getY() > toY) {
+            while (robot.getDirection() != Direction.DOWN) {
+                robot.turnLeft();
+            }
+        }
+        while (robot.getY() != toY) {
+            robot.stepForward();
+        }
     }
-}
 
+    //your code
```

Comment:
let's remove commented code and unneeded empty lines

Reply:
Done